app/src/ActionModal/components/UndelegationModal.vue

```diff
@@ -462,13 +459,12 @@ export default {
       variables() {
         return {
           networkId: this.network,
-          address: this.address,
-          denom: this.stakingDenom,
+          address: this.address
         }
       },
       /* istanbul ignore next */
       update(data) {
-        return data.balance || { amount: 0 }
+        return data.balancesV2.find(({denom}) => this.stakingDenom) || { total: 0, available: 0 }
```

Comment:
What are these changes? :eyes:

Reply:
ups. will remove, when you tested. that was from fixing the undelegation modal stake displaying

Reply:
OK, so I should test this PR as it is right now? To confirm your hypothesis I guess

Reply:
you can, but if the extension works for you know, we can just close it

Reply:
Today it is not working again. Let me try this branch

Reply:
with this branch I don't get to see any address at all:

![image](https://user-images.githubusercontent.com/40721795/92716488-2bc51380-f35f-11ea-9d36-69129d413262.png)

Reply:
oh I see. I need to rebuild extension...

Reply:
yes, it works!! :tada: 

![image](https://user-images.githubusercontent.com/40721795/92717437-785d1e80-f360-11ea-9bfd-b3f9e244117e.png)